mergin/client.py

```diff
@@ -287,6 +287,47 @@ def create_project_and_push(self, project_name, directory, is_public=False):
             if mp.inspect_files():
                 self.push_project(directory)
 
+    def paginated_projects_list(self, tags=None, user=None, flag=None, q=None, page=1, per_page=50):
+        """
+        Find all available mergin projects.
+
+        :param tags: Filter projects by tags ('valid_qgis', 'mappin_use', input_use')
+        :type tags: List
+
+        :param user: Username for 'flag' filter. If not provided, it means user executing request.
+        :type user: String
+
+        :param flag: Predefined filter flag ('created', 'shared')
+        :type flag: String
+
+        :param q: Search query string
+        :type q: String
+
+        :param page: Page number for paginated projects list
+        :type page: Integer
+
+        :param per_page: Number of projects fetched per page, max 100 (restriction set by server)
+        :type per_page: Integer
+
+        :rtype: List[Dict]
+        """
+        params = {}
+        if tags:
+            params["tags"] = ",".join(tags)
+        if user:
+            params["user"] = user
+        if flag:
+            params["flag"] = flag
+        if q:
+            params["q"] = q
+        params["order_by"] = 'namespace'
+        params["descending"] = False
```

Comment:
it would be good to have the ordering configurable (+ document what are the possible ordering values)

Reply:
Done for `order_by` which is now deprecated and also for the shiny `order_params`.